README.md

```diff
@@ -14,14 +20,55 @@ The QFSAE team designs its boards using Altium. If you do not have Altium Design
+## Installation & Setup
+Altium is licensed software. In order to get a license, contact the electrical lead (Logan - Q21) to get your email added to the organization. Once you've been added, you should receive an email with your login. It is recommended that you sign in to [altium.com](https://www.altium.com/) and change your password. 
+
+### Download
+Altium is available for download on their [downloads page](https://www.altium.com/products/downloads). You will be prompted to log in, so make sure you have your account set up first! Download the lastest version (20.x.x - Q21). Most versions are backwards compatible, so it is not essential to stay completely up to date but it is recommended to always use the latest major release.
+
+Default installation settings are recommended.
+
+### Using a license 
+Before you can use Altium you must first sign into your account and check out one of the organization's seats. This can be done by clicking on the "Not Signed In" dropdown in the top right of the window and choosing "Sign in":
+
+![Sign in Window](https://i.imgur.com/ysroxVE.png)
+
+Once signed in, you must choose a license to take out in the License Management window. You can access this window by clicking "Licenses" from the same spot as shown above. To use a license, select it and click on the Use button below it. Make sure that the license's status is OK, and there are still unassigned seats. The team has 10 seats for now, so only 10 people can use the license at the same time. 
+
+![License Panel](https://i.imgur.com/Kl5esJi.png)
+
+Please try to release your license whenever you are not working in Altium we don't run out of seats! 
+
+*Note:* If you are having issues using a license, verify your system time is correct. This issue is common on Windows/Linux dual boots where rebooting back into Windows can offset the time. 
+
+### Part Library Setup
+For the majority of projects the only library required is the JLCPCB Altium component library. It can be located in the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, under **"JLCSMT_LIB"**. 
+
+#### Adding Parts Libraries
+The JLC library is a linked repository, so start by navigating to the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, clicking through to the JLC library repository and downloading it. Save the library in a safe directory (ie. not Downloads!) We will be adding it to your Altium installation so it is available for use in all projects.
```

Comment:
No need to have people explicitly download the linked library repository as we can manage it through sub modules. People will have the library downloaded by running `git submodule update --init` after they clone this repository. The same applies for Celestial library files as well.